src/Common/src/Interop/Linux/cgroups/Interop.cgroups.cs

```diff
@@ -79,6 +85,16 @@ private static bool TryReadMemoryValueFromFile(string path, out ulong result)
                         result = checked(ulongValue * multiplier);
                         return true;
                     }
+
+                    var encoding = new UTF8Encoding();
+                    string value = encoding.GetString(bytes);
+                    if ("max".Equals(value, StringComparison.InvariantCultureIgnoreCase))
```

Comment:
cgroupv2 uses explicit values or `max` to indicate there's no memory limit active. For example:

```
$ cat /sys/fs/cgroup/user.slice/user-1000.slice/user@1000.service/gnome-terminal-server.service/memory.max 
max
```

This code here treats that as not having a memory limit defined, so the caller (`Process.GetWorkingSetLimits`) can fall back to reading `/proc/self/stat`.

Reply:
Can this be OrdinalIgnoreCase rather than InvariantCultureIgnoreCase?

Reply:
Yes. We can probably even do plain `Ordinal`, `max` is the only stringthat the specification asks for.